### PR TITLE
Concept entry: rust comments

### PR DESCRIPTION
**Pull request by cdcalderontenorio**
### Description

- What? A new entry: Rust, comments
- Why? It is one of the tasks for the course Learn Git & GitHub
- How? I looked up information on the internet, read different blogs and compare the information according to my own experience with Java

### Type of Change

- Adding a new entry

### Checklist

- [X ] All writings are my own.
- [X ] My entry follows the Codecademy Docs style guide.
- [X ] My changes generate no new warnings.
- [X ] I have performed a self-review of my own writing and code.
- [X ] I have checked my entry and corrected any misspellings.
- [X ] I have made corresponding changes to the documentation if needed.
- [X ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/rust/concepts/comments/comments.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Comments'
+Description: 'A comment is a a line, or a few lines, of text which add an extra explanation to the code to ease readability of the code for people.'
```
Comment (ishg-153) [inline content/rust/concepts/comments/comments.md:3]:
```suggestion
Description: 'A comment is a line, or a few lines, of text which adds an extra explanation to the code to ease readability of the code for people.'
```

File: content/rust/concepts/comments/comments.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Tags:
+  - 'Best Practices'
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'rust-for-programmers'
+  - 'paths/computer-science'
+---
+
+A **comment** is a line, or few lines, of text which add extra explanation to the code to ease readability of the code for people. Any comment will be ignored and not executed by the compiler. In Rust programming, there are two types of comments:
```
Comment (ishg-153) [inline content/rust/concepts/comments/comments.md:15]:
```suggestion
A **comment** is a line, or few lines, of text that adds extra explanation to the code to ease readability of the code for people. Any comment will be ignored and not executed by the compiler. In Rust programming, there are two types of comments:
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@cdcalderontenorio I have suggested two changes. Please make those and let me know when you are done.

File: content/rust/concepts/comments/comments.md (hunk 2 above)
Comment (cdcalderontenorio) [reply content/rust/concepts/comments/comments.md:15]: Hi, thanks. Changes done.

File: content/rust/concepts/comments/comments.md (hunk 1 above)
Comment (cdcalderontenorio) [reply content/rust/concepts/comments/comments.md:3]: Hi, thanks. Changes done.

File: content/rust/concepts/comments/comments.md (hunk 2 above)
Comment (cdcalderontenorio) [reply content/rust/concepts/comments/comments.md]: Do I need to open a new PR?

Comment (ishg-153) [reply content/rust/concepts/comments/comments.md]: Nope, there's no need for a new PR. We usually suggest changes if required; it is expected that you make them in the same branch and PR.

**Review by ishg-153 — APPROVED** [review]
@cdcalderontenorio Great! This looks ready for a second review.

File: content/rust/concepts/comments/comments.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Comments'
+Description: 'A comment is a line, or a few lines, of text that adds an extra explanation to the code to ease readability of the code for people.'
```
Comment (SSwiniarski) [inline content/rust/concepts/comments/comments.md:3]:
```suggestion
Description: 'A comment is a line, or a few lines, of text that adds an extra explanation to the code to ease readability of the code.'
```

File: content/rust/concepts/comments/comments.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Tags:
+  - 'Best Practices'
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'rust-for-programmers'
+  - 'paths/computer-science'
+---
+
+A **comment** is a line, or few lines, of text that adds extra explanation to the code to ease readability of the code for people. Any comment will be ignored and not executed by the compiler. In Rust programming, there are two types of comments:
```
Comment (SSwiniarski) [inline content/rust/concepts/comments/comments.md:15]:
```suggestion
A **comment** is a line, or few lines, of text that adds extra explanation to the code to ease readability of the code. Any comment will be ignored and not executed by the compiler. In Rust programming, there are two types of comments:

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cdcalderontenorio I did a second review and left a couple of comments. Tag me when you've addressed them.

**Review by SSwiniarski — APPROVED** [review]
@cdcalderontenorio Looks good to merge.

**Pull request merged**